compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -1062,6 +1062,84 @@ static LogicalResult setTransposeConfig(func::FuncOp entryPoint,
       workgroupSize);
 }
 
+/// Set the configuration for argmax that can be mapped to argmax uKernel.
+/// Distribute all parallel dim across different wg, and only use single
```

Comment:
Please spell out "wg" in comments--the purpose of comments is to make the code easier to understand. Having various abbr. makes that less obvious. :)

Reply:
Done, Thanks, this makes sense! :)